src/createServer.js

```diff
@@ -1,3 +1,57 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    const [text, queryString] = req.url.slice(1).split('?');
+    const queryParams = new URLSearchParams(queryString);
+    const toCase = queryParams.get('toCase');
+
+    if (!text || !toCase || !cases.includes(toCase)) {
```

Comment:
Instead of duplicating check like `!cases.includes(toCase)`, just check for `errors.length` afterwards
```suggestion
```